package.json

```diff
@@ -1,26 +1,31 @@
 {
   "name": "crowdbotics-modules",
-  "version": "1.0.0",
+  "version": "1.1.0",
   "description": "# React Native Scaffold",
   "main": "index.js",
+  "bin": {
+    "crowdbotics-modules": "./index.js"
+  },
   "type": "module",
   "engines": {
-    "node": ">14.0"
+    "node": ">16"
```

Comment:
Is the intention to require versions of at least 17?

Reply:
Good catch, thanks. This was meant to require 16 *and* above. Fixed it in 08d8c56.